### ISSUE TITLE
Timeline Chart Y-axis Grid Lines

### ISSUE THREAD

**Issue by JDHealy17** [apexcharts/apexcharts.js#1164]
What's happening: the number of y-axis grid lines being displayed matches the number of data-series plotted. These lines do not line up with y-axis ticks/labels if you use forceNiceScale. There is no option in the docs to specify the number of lines or to align them with the axis ticks.

What I'd like to see: have the grid lines auto-align with the axis ticks. For my application, it appears to be working fine for the x-axis values, but the y-axis labels will only align if I allow it to scale "ugly".

**Comment by JDHealy17** [comment]
Update: it appears this issue affects "datetime" x-axis type, but not "category" x-axis type

**Comment by junedchhipa** [comment]
Can you please post a codepen?
I cannot reproduce it.

**Comment by JDHealy17** [comment]
> Can you please post a codepen?
> I cannot reproduce it.

Here is a [codepen](https://codepen.io/jdhealy17/pen/NWPgZPy) I modified from one of yours.
You'll note: 
1) X-axis ticks and labels are not actually 'on,' but 'between,' despite the code.
2) Watch what happens to the y-axis (ticks and labels), x-axis labels, and the data labels when you change xaxis.type from datetime to category.
3) While xaxis.type is datetime, add another (can copy/paste, but change x-value date) x-y-fillcolor set in the series. Watch as another background grid line is added. Changing back to xaxis.type as category, it will resume following the tickAmount and forceNiceScale properties.
4) While you're looking at it, note how the tooltip custom formatter is set to tooltip.x, despite modifying the display of the y values. Go ahead and try cutting that out and seeing what happens, and also try adding it back under tooltip.y. 

Let me know if I can elaborate any more, these are the issues I've noticed while working on the example, all mirrored in my personal code. Thanks!
